examples/with-iron-session/.env.development

```diff
@@ -0,0 +1,6 @@
+# ⚠️ The SECRET_COOKIE_PASSWORD should never be inside your repository directly, it's here only to ease
```

Comment:
Can we name these `.env.development.local`? We can exclude the local versions from our repository's `.gitignore` for the examples directory.

Reply:
Do you mean you would prefer to not have those files committed to the example code? If I name the file .env.development.local then it will be gitignored and the example deploy won't work anymore.

Before this commit the secret key was directly inside the package.json.

Let me know what you want

Reply:
Oh maybe you meant: we can rename it, allow them inside the examples here but they will be gitignored if someone copy the code locally, is that what you meant?